lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -143,13 +143,27 @@ def get_string_for_issue(issue)
       encapsulated_title = encapsulate_string issue["title"]
 
       title_with_number = "#{encapsulated_title} [\\##{issue['number']}](#{issue['html_url']})"
+      unless @options[:issue_line_labels].empty?
+        title_with_number = issue_line_with_labels(title_with_number, issue)
+      end
       issue_line_with_user(title_with_number, issue)
     end
 
     private
 
+    def issue_line_with_labels(line, issue)
+      labels = if options[:issue_line_labels] == ["ALL"]
+                 issue["labels"]
+               else
+                 issue["labels"].select { |label| options[:issue_line_labels].include? label["name"] }
+               end
+      labels_string = labels.map { |label| " \[[#{label['name']}](#{label['url'].sub! 'api.github.com/repos', 'github.com'})\]" }.join("")
+      line = "#{line}#{labels_string}"
+      line
```

Comment:
@olleolleolle, this all works for me. Thanks for making the code changes (I'm not a Ruby developer, but this project has been very useful). I've updated the code for this issue and rebased.